sig-advocacy-and-outreach/summer-ospp/README_zh-CN.md

```diff
@@ -13,19 +13,19 @@ KubeSphere 参与[开源软件供应链点亮计划 - 暑期 2021](https://summe
-| [Notification Manager 发送通知到 pushover](support-send-notifications-to-pushover_zh-CN.md)<br/>使 Notification Manager 可以发送通知消息到 pushover。<br/>导师：[benjaminhuo](https://github.com/benjaminhuo),[lei](https://github.com/wanjunlei) | Notification Manager | Golang，REST API，Kubernetes                        | 中 |
-| [集成 Gatekeeper](kubesphere-gatekeeper-integration_zh-CN.md) <br/>将 Gatekeeper 作为准入策略引擎集成到 KubeSphere。<br/>导师：[Hongming](https://github.com/wansir/), [Roland](https://github.com/rolandma1986/)| Security | Golang，React，OPA，Gatekeeper，Kubernetes | 中 |
-| [集成 SAML 认证](KubeSphere-SAML-integrations_zh-CN.md) <br/>集成 SAML 2.0 认证协议到 KubeSphere<br/>导师：[Hongming](https://github.com/wansir/), [Roland](https://github.com/rolandma1986/)| Security | Golang，REST API/OpenAPI，SAML | 低 |
-| [OpenFunction 的函数构建器开发](OpenFunction-function-builder_zh-CN.md) <br>开发并优化函数构建器，使其可以识别 Python、Nodejs、Java、PHP（可选）、Rust（可选）语言，并将函数构建成可以运行在 OpenFunction 中的应用镜像。<br>导师：[Benjamin Huo](https://github.com/benjaminhuo)、[Fang Tian](https://github.com/tpiperatgod/) | Functions-as-a-Service | Golang，Tekton Cloud Native Buildpacks，OpenFunction | 中 |
-| [OpenFunction 的 nodejs 函数框架开发](OpenFunction-function-framework-nodejs_zh-CN.md) <br>开发用于处理 nodejs 代码的函数框架，为用户的业务代码提供适用于云原生环境的多种工具套件。<br>导师：[Benjamin Huo](https://github.com/benjaminhuo)、[Fang Tian](https://github.com/tpiperatgod/) | Functions-as-a-Service | Golang，Tekton Cloud Native Buildpacks，OpenFunction，Nodejs | 中 |
-| [OpenFunction 项目官网开发](openfunction-website_zh-CN.md)<br>在项目导师的帮助下，为全新的 Serverless 开源项目 [OpenFunction](https://github.com/OpenFunction/OpenFunction) 从零开始搭建与开发项目的官方网站，并完成项目上线。<br>导师：[Feynman Zhou](https://github.com/FeynmanZhou)、[Sherlock](https://github.com/Sherlock113) | Website | JavaScript，HTML，CSS，Markdown | 低 |
-| [KubeSphere 项目官网优化](kubesphere-website_zh-CN.md)<br>在项目导师的帮助下，为开源项目 [KubeSphere](https://github.com/kubesphere/kubesphere) 完善 KubeSphere 官网以及文档网站，实现部分功能特性发布到线上。<br>导师：[Bo Liu](https://github.com/liuboaibc)、[Feynman Zhou](https://github.com/FeynmanZhou) | Website | JavaScript，HTML，CSS，Markdown | 低 |
-| [KubeSphere 官网文档写作与本地化](document-localization_zh-CN.md)<br>在项目导师的帮助下，为开源项目 [KubeSphere](https://github.com/kubesphere/kubesphere) 撰写新的产品文档，根据用户需求为官方文档的本地化，将英文文档翻译成中文文档。<br>导师：[Feynman Zhou](https://github.com/FeynmanZhou)、[Sherlock](https://github.com/Sherlock113) | Document | 优秀的计算机领域知识/英语书写与翻译能力，Markdown | 低 |
+| 项目 | 领域 | 技能 | 难 度 | 学生 |
+| --- | --- | --- | --- | --- |
+| [kubeSphere Jenkins 客户端重构](kubeSphere-jenkins-client-refactor_zh-CN.md) <br/>从 [KubeSphere](https://github.com/kubesphere/kubesphere/) 核心代码中的 DevOps 部分将 Jenkins 客户端代码分离开，或者，使用已有的 Jenkins 客户端。<br/>导师： [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/) | DevOps | Golang，REST API/OpenAPI，Jenkins，Kubernetes | 中 | 吴晓涵 |
```

Comment:
It would be better if you can put the GitHub profile link of the students here. But this is only a suggestion. I don't block the review on this.

Reply:
I intend to let each student supplement it by himself